fe/fe-core/src/main/java/com/starrocks/sql/optimizer/MvRewritePreprocessor.java

```diff
@@ -230,7 +229,7 @@ private LogicalOlapScanOperator createScanMvOperator(MaterializationContext mvCo
         // Construct partition/distribution key column refs to filter conjunctions which need to retain.
         Set<String> mvPruneKeyColNames = Sets.newHashSet();
         distributedColumns.stream().forEach(distKey -> mvPruneKeyColNames.add(distKey.getName()));
-        mv.getPartitionNames().stream().forEach(partName -> mvPruneKeyColNames.add(partName));
+        mv.getPartitionColumnNames().stream().forEach(partName -> mvPruneKeyColNames.add(partName));
```

Comment:
Why change it from `getPartitionNames` to `getPartitionColumnNames()` ?

Reply:
the logic is to find the partition related predicates, should use partition column names to find them, not partition names.